internal/bgp/bgpfrr/config.go

```diff
@@ -0,0 +1,121 @@
+package bgpfrr
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"text/template"
+)
+
+// TODO configFileName will need to be set to the filename of the FRR configuration
+// file in the container.
+var configFileName = "frr.conf"
+
+const configTemplate = `
+log stdout {{.Loglevel}}
+hostname {{.Hostname}}
+
+{{range .Routers -}}
+router bgp {{.MyASN}}
+  no bgp default ipv4-unicast
+  bgp router-id {{.RouterId}}
+{{range .Neighbors }}
+  neighbor {{.Addr}} remote-as {{.ASN}}
+{{- end }}
+{{range $n := .Neighbors -}}
```

Comment:
There's an AF block per Kubernetes Service. I may be missing out on something trivial but it seems this could be optimized?

```
/ # cat /etc/frr/frr.conf 

log stdout 
hostname 

router bgp 64512
  no bgp default ipv4-unicast
  bgp router-id <nil>

  neighbor 10.89.0.5:179 remote-as 64512

  address-family ipv4 unicast
    neighbor 10.89.0.5:179 activate
    network 192.168.10.0/32
  exit-address-family
  address-family ipv4 unicast
    neighbor 10.89.0.5:179 activate
    network 192.168.10.1/32
  exit-address-family
```

```
$ kubectl get svc
NAME         TYPE           CLUSTER-IP      EXTERNAL-IP    PORT(S)                                  AGE
kubernetes   ClusterIP      10.96.0.1       <none>         443/TCP                                  31m
mirror       LoadBalancer   10.96.121.174   192.168.10.0   80:31853/TCP,81:32147/TCP,82:30054/TCP   2m50s
mirror-bkp   LoadBalancer   10.96.161.133   192.168.10.1   80:32409/TCP,81:31045/TCP,82:31755/TCP   2m50s
```

Maybe something like this below?

```
  address-family ipv4 unicast
    neighbor 10.89.0.5:179 activate
    network 192.168.10.0/32
    network 192.168.10.1/32
  exit-address-family
```

Reply:
Yeah, I absolutely think this is the case. I did consider this but it was just easier to do it in the less elegant way we are currently doing. As I mentioned in another comment, I don't think that the configuration that we are rendering in this PR is optimal. However, until we have a working e2e CI, I didn't want to focus on that too much as it will be much easier to refactor at that point. Feel free to modify it if you would like, if not. I can address in the future.